docs/machineregistration-reference.md

```diff
@@ -104,6 +104,45 @@ The only required value for a successful installation is the `device` key as we
   ```
 </details>
 
+#### config.elemental.reset
+
+Contains the reset configuration that would be applied via `elemental-register --reset`, when booted from the recovery partition and passed to [`elemental reset`](https://github.com/rancher/elemental-toolkit/blob/main/docs/elemental_reset.md)
+
+Supports the following values:
+
+| Key         | Type   | Default value | Description                                                                                                                                |
+|-------------|--------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| enabled           | bool   | false  | Allow reset on all registered machines  |
```

Comment:
I would be more explicit that the reset is just applied to the created MachineInventories (to avoid misunderstanding that the reset functionality can be dynamically controlled on existing machines from the MachineRegistration). Something like:

```suggestion
| enabled           | bool   | false  | MachineInventories created from this MachineRegistration will have reset functionality enabled |
```